docs/content/how-to/sdn/other-sdn-providers.md

```diff
@@ -0,0 +1,402 @@
+    customresourcedefinition.apiextensions.k8s.io/tigerastatuses.operator.tigera.io created
+    namespace/calico-apiserver created
+    namespace/calico-system created
+    namespace/tigera-operator created
+    apiserver.operator.tigera.io/default created
+    installation.operator.tigera.io/default created
+    configmap/calico-resources created
+    clusterrolebinding.rbac.authorization.k8s.io/tigera-operator created
+    clusterrole.rbac.authorization.k8s.io/tigera-operator created
+    serviceaccount/tigera-operator created
+    deployment.apps/tigera-operator created
+    ~~~
+
+### Checks
+
+We should see the following pods running in the `tigera-operator` namespace:
+
+  ~~~sh
+  oc -n tigera-operator get pods
+  ~~~
+
+  ~~~output
+  NAME                              READY   STATUS    RESTARTS   AGE
+  tigera-operator-dc7c9647f-fvcvd   1/1     Running   0          2m1s
+  ~~~
+
+We should see the following pods running in the `calico-system` namespace:
+
+  ~~~sh
+  oc -n calico-system get pods
+  ~~~
+
+  ~~~output
+  NAME                                       READY   STATUS    RESTARTS   AGE
+  calico-kube-controllers-69d6d5ff89-5ftcn   1/1     Running   0          2m1s
+  calico-node-6bzth                          1/1     Running   0          2m2s
+  calico-node-bl4b6                          1/1     Running   0          2m2s
+  calico-typha-6558c4c89d-mq2hw              1/1     Running   0          2m2s
+  csi-node-driver-l948w                      2/2     Running   0          2m1s
+  csi-node-driver-r6rgw                      2/2     Running   0          2m2s
+  ~~~
+
+We should see the following pods running in the `calico-apiserver` namespace:
+
+  ~~~sh
+  oc -n calico-apiserver get pods
+  ~~~
+
+  ~~~output
+  NAME                                READY   STATUS    RESTARTS   AGE
+  calico-apiserver-7bfbf8fd7c-d75fw   1/1     Running   0          84s
+  calico-apiserver-7bfbf8fd7c-fqxjm   1/1     Running   0          84s
+  ~~~
+
+The nodes should've moved to `Ready` state:
+
+  ~~~sh
+  oc get nodes
+  ~~~
+
+  ~~~output
+  NAME             STATUS   ROLES    AGE   VERSION
+  hosted-worker1   Ready    worker   10m   v1.27.8+4fab27b
+  hosted-worker2   Ready    worker   10m   v1.27.8+4fab27b
+  ~~~
+
+The HostedCluster deployment will continue, at this point the SDN is running.
+
+## Cilium
+### Deployment
+
+In this scenario we are using the Cilium version v1.14.5 which is the last one at the time of this writing. The steps followed rely on the [docs](https://docs.cilium.io/en/stable/installation/k8s-install-openshift-okd/) by Cilium project to deploy Cilium on OpenShift.
+
+1. Create a `HostedCluster` and set its `HostedCluster.spec.networking.networkType` to `Other`.
+
+2. Wait for the HostedCluster's API to be ready. Once it's ready, get the admin kubeconfig.
+
+3. Wait for compute nodes to join the cluster. Keep in mind since the SDN is not deployed yet, they will remain in `NotReady` state.
```

Comment:
Same as above, no need to wait for nodes.

Reply:
fixed